kodilla-good-patterns/src/main/java/com/kodilla/goodPatterns/challenges/MovieStoryRun.java

```diff
@@ -0,0 +1,24 @@
+package com.kodilla.goodPatterns.challenges;
+import com.kodilla.goodPatterns.challenges.MovieStory;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class MovieStoryRun {
+    public static void main(String[] args){
+        MovieStory movieStory = new MovieStory();
+
+        String movieStoryList = movieStory.getMovies().entrySet().stream()
+                .map(k -> k.getValue())
+                .map(l ->l.toString())
```

Comment:
powinienes uzyc `flatMap(s->s.stream())`

Reply:
nie działa mi tak. wywala cannot metod resolve stream

Reply:
to dalej nie jest to

wlasciwe rozwiazanie to
```
String movieStoryList = movieStory.getMovies().entrySet().stream()
                .map(k -> k.getValue())
                .flatMap(e->e.stream())
                .collect(Collectors.joining("!","<<", ">>"));